{{template "header.html".}}
{{template "app/templates/navbar.html".}}
<div class="relative mt-8 py-10 bg-white overflow-hidden">
	<div class="relative px-4 sm:px-6 lg:px-8">
		<div class="max-w-screen-xl mx-auto">
			<div class="mb-4">
				<h1 class="mt-8 mb-2">
					<span class="mt-8 block text-4xl leading-8 font-semibold text-gray-900 sm:text-4xl">
						Axolotl Nomenclature
					</span>
				</h1>
				<hr>
			</div>
			<h2 class="mt-8 text-2xl font-semibold">
				Axolotl gene and transgenic nomenclature guidelines
			</h2>
			<p class="mt-2">
				These guidelines are based on Nowoshilow et al, <a
					href="https://anatomypubs.onlinelibrary.wiley.com/doi/10.1002/dvdy.351">Developmental
					Dynamics</a>, 2021. Last updated on 25.04.2021 by Prayag Murawala. For suggestions on guidelines or
				to propose a new gene name, please write an email to <a class="text-axo-green" href="mailto:info@axobase.org">info@axobase.org</a>.
			</p>
			<h2 class="mt-8 mb-4 text-2xl font-semibold">
				Axolotl gene and transgenic nomenclature guidelines
			</h2>
			<p>
				<a class="text-axo-green" href="https://www.genenames.org/about/guidelines/">Human</a>,
				<a class="text-axo-green" href="http://www.informatics.jax.org/mgihome/nomen/">Mice</a>,
				<a class="text-axo-green"
					href="https://zfin.atlassian.net/wiki/spaces/general/pages/1818394635/ZFIN+Zebrafish+Nomenclature+Conventions#ZFINZebrafishNomenclatureConventions-7">Zebrafish</a>,
				<a class="text-axo-green" href="https://www.xenbase.org/gene/static/geneNomenclature.jsp">Xenopus</a>,
				<a class="text-axo-green" href="http://birdgenenames.org/cgnc/guidelines">Chicken</a>,
				<a class="text-axo-green" href="https://wiki.flybase.org/wiki/FlyBase:Nomenclature">Drosophila</a>
			</p>
			<h2 class="mt-8 mb-4 text-2xl font-semibold">Contents:</h2>
			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#a-quick-summary-of-nomenclature">A quick summary of
					nomenclature</a>
			</p>
			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#a-detailed-description-of-nomenclature">A detailed
					description of nomenclature</a>
			</p>
			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#species-symbol">Species Symbol</a>
			<p class="ml-8 mb-2">1. Species abbreviation</p>
			</p>

			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#gene-mrna-cdna-protein">Gene/mRNA/cDNA/Protein</a>
			<p class="ml-8 mb-2">2. Gene</p>
			<p class="ml-8 mb-2">3. Protein</p>
			</p>

			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#gene-nomenclature-in-axolotl">Gene nomenclature in
					axolotl</a>
			<p class="ml-8 mb-2">4. Orthologs and Paralogs</p>
			<p class="ml-8 mb-2">5. Novel genes</p>
			<p class="ml-8 mb-2">6. Transcript variants</p>
			<p class="ml-8 mb-2">7. Non-coding transcripts</p>
			<p class="ml-8 mb-2">8. Mitochondrial genes</p>
			</p>

			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#chromosomes-and-aberrations">Chromosomes and
					aberrations</a>
			<p class="ml-8 mb-2">9. Deficiencies, duplication, inversion, insertion, translocation</p>
			</p>

			<p class="mb-2">
				<a class="text-axo-green" href="nomb-2menclature#transgenic-lines-and-constructs">Transgenic lines and
					constructs</a>
			<p class="ml-8 mb-2">10. Random insertions
			<p>
			<p class="ml-8 mb-2">11. Knockout lines
			<p>
			<p class="ml-8 mb-2">12. Knock-in lines
			<p>
			</p>

			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#members-of-the-axolotl-gene-nomenclature-committee">Members
					of the axolotl gene nomenclature committee</a>
			</p>
			<p class="mb-2">
				<a class="text-axo-green" href="nomenclature#references">References</a>
			</p>
			<h3 id="a-quick-summary-of-nomenclature" class="mt-8 mb-4 text-xl font-semibold">
				Summary of Nomenclature:
			</h3>
			<p class="mb-2">
				Species symbol: Amex
			</p>
			<p class="mb-2">
				Gene/mRNA/cDNA: <i>Prrx1</i>
			</p>
			<p class="mb-2">
				Protein: PRRX1
			</p>
			<p class="mb-2">
				Paralogs: <i>Prrx1</i> and <i>Prrx2</i>
			</p>
			<p class="mb-2">
				Transcript variant: <i>Prrx1.1</i>, <i>Prrx1.2</i>
			</p>
			<p class="mb-2">
				Transgenic line: tgSceI(Mmu<i>.Prrx1:GFPnls-T2a-ERT2-Cre-ERT2</i>
				)<sup>Labcode</sup>, tgSceI(Mmu
				<i>.Prrx1:GFPnls-T2a-ER<sup>T2</sup>-Cre-ER<sup>T2</sup>;CAGGS:loxP-GFP-loxP-Cherry</i>
				)<sup>Labcode</sup>
			</p>
			<p class="mb-2">
				Gene knockout: tm(Prrx1<sup>153v6D8/153v6D8</sup>)<sup>Labcode</sup>,
				tm(Prrx1<sup>153v6I5/+</sup>)<sup>Labcode</sup>
			</p>
			<p class="mb-2">
				Gene knock-in: tm(Prrx1<sup>t/+</sup>:Prrx1-T2a-Cherry)<sup>Labcode</sup>,
				tm(Prrx1r/+:Cherry)<sup>Labcode</sup>, tm(Prrx1r/+:miniPrrx1-T2a-Cherry)<sup>Labcode</sup>
			</p>
			<h3 id="a-detailed-description-of-nomenclature" class="mt-8 mb-4 text-xl font-semibold">
				Detailed description of nomenclature:
			</h3>
			<h4 id="species-symbol" class="mt-4 mb-4 text-lg font-semibold">
				Species symbol
			</h4>
			<p class="mb-2">
			<ol class="list-decimal list-inside">
				<li>
					Species Abbreviation<br>
					<p class="ml-6 mt-2">
						Although the axolotl (*Ambystoma mexicanum*) is one of the most commonly used
						salamanders, a number of other species such as *Ambystoma tigrinum*
						(Tiger salamander), *Ambystoma maculatum* (Spotted salamander) *Notophthalmus
						viridescens* (Eastern newt), and *Pleurodeles waltl* (Iberian newt) are
						commonly used for tetrapod tissue regeneration research<sup>39</sup>. A
						comparative-omics study often requires identification of orthologous genes
						across salamander species. We recommend using the first letter denoting the
						genus name in upper case and the first three letters of the species name in
						lower case regular font to create the species-specific acronym. For
						<i><u>A</u>mbystoma</i> <i><u>mex</u>icanum</i>, we recommend using Amex as a
						species symbol. For other ambystomatid species, the same convention would
						follow, such as Amac (*A. maculatum*), Aand (*A. andersoni*), and Acal
						(*A. californiensi*). If there is a subspecies, the first letter of the
						subspecies name can be assigned as a fifth letter in a lower-case italics, and
						a sixth letter to differentiate between subspecies that share the same first
						letter of the subspecies name. For example, there are several *A. tigrinum*
						subspecies, including *A. tigrinum mavortium*(Atigma), *A. tigrinum
						melanosticum*(Atigme), and *A. tigrinum nebulosum*(Atign).
					</p>
				</li>
			</ol>
			</p>
			<h4 id="gene-mrna-cdna-protein" class="mt-4 mb-4 text-lg font-semibold">
				Gene/mRNA/cDNA/Protein
			</h4>
			<p class="mb-2">
			<ol class="list-decimal list-inside" start=2>
				<li>
					Gene
					<p class="ml-6 mt-2">
						A gene symbol is described by a series of alphanumeric characters, where the
						first character is upper-case italics and the rest are lower-case italics.
						This is in line with the mouse gene nomenclature and it follows current
						axolotl gene annotations<sup>2</sup>. For example, a gene Paired related
						homeobox 1 is referred to as *Prrx1*.
					</p>
					<p class="ml-6 mt-4">2.1 Species Ambiguity</p>
					<p class="ml-6 mt-2">
						When there is an ambiguity about the orthology of a gene from a
						species, we recommend using a prefix to indicate the species name. The
						prefixes should not be considered part of the name and hence, a period should
						be placed between species and the gene name, while an entire phrase should be
						written in italics, e.g. Amex.*Prrx1*. When there is no ambiguity of
						species used for the study, the species symbol must be omitted when
						describing the gene.
					</p>
					<p class="ml-6 mt-4">2.2 Gene, mRNA, cDNA</p>
					<p class="ml-6 mt-2">
						Often a gene, mRNA and cDNA need to be distinguished in a text.
						In such cases, we recommend putting “gene”, “mRNA” or “cDNA” in regular font
						in parentheses in front of the gene symbol. Gene, mRNA or cDNA words must be
						omitted to describe a gene when there is no ambiguity or after describing
						once in the beginning of the communication.
					</p>
					<p class="ml-6 mt-4">Gene: <i>Prrx1</i></p>
					<p class="ml-6 mt-2">
						In case of ambiguity between gene, mRNA, cDNA: (gene)*Prrx1*, (mRNA)*Prrx1*, (cDNA)*Prrx1*
					</p>
					<p class="ml-6">
						In case of species ambiguity: (gene)Amex. *Prrx1*
					</p>
				</li>
				<li class="mt-8">
					Protein
					<p class="ml-6 mt-2">
						The protein symbol is same as the gene symbol, but in regular (non-italic) font, with all
						characters in upper-case.
					</p>
					<p class="ml-6 mt-2">
						Protein: PRRX1
					</p>
				</li>
			</ol>
			</p>
			<h4 class="mt-4">
				<span class="font-semibold">Table1:</span> Gene and protein naming convention in axolotl,
				zebrafish, <i>Xenopus</i>, anole, mouse, chicken, and human.
			</h4>
			<table class="mt-8 mb-8 table-fixed border-separate border border-green-800">
				<thead>
					<tr>
						<th class="p-2 border border-green-600">Species</th>
						<th class="p-2 border border-green-600">Gene nomenclature</th>
						<th class="p-2 border border-green-600">Protein nomenclature</th>
						<th class="p-2 border border-green-600">Reference</th>
					</tr>
				</thead>
				<tbody>
					<tr>
						<td class="p-2 border border-green-600">Axolotl</d>
						<td class="p-2 border border-green-600"><i>Prrx1</i></d>
						<td class="p-2 border border-green-600"><i>PRRX1</i></d>
						<td class="p-2 border border-green-600">Nowoshilow et al, Dev Dyn, 2021</d>
					</tr>
					<tr>
						<td class="p-2 border border-green-600">Zebrafish</td>
						<td class="p-2 border border-green-600"><i>prrx1<i></td>
						<td class="p-2 border border-green-600"><i>Prrx1</i></td>
						<td class="p-2 border border-green-600">Reference<sup>28,29</sup></td>
					</tr>
					<tr>
						<td class="p-2 border border-green-600">Xenopus</td>
						<td class="p-2 border border-green-600"><i>prrx1</i></td>
						<td class="p-2 border border-green-600"><i>Prrx1</i></td>
						<td class="p-2 border border-green-600">Reference<sup>26,27</sup></td>
					</tr>
					<tr>
						<td class="p-2 border border-green-600">Anole</td>
						<td class="p-2 border border-green-600"><i>prrx1</i></td>
						<td class="p-2 border border-green-600"><i>PRRX1</i></td>
						<td class="p-2 border border-green-600">Referenc<sup>32</sup></td>
					</tr>
					<tr>
						<td class="p-2 border border-green-600"> Mouse</td>
						<td class="p-2 border border-green-600"> <i>Prrx1</i></td>
						<td class="p-2 border border-green-600"> <i>PRRX1</i></td>
						<td class="p-2 border border-green-600"> Reference<sup>40</sup></td>
					</tr>
					<tr>
						<td class="p-2 border border-green-600">Chicken</td>
						<td class="p-2 border border-green-600"><i>PRRX1</i></td>
						<td class="p-2 border border-green-600"><i>PRRX1</i></td>
						<td class="p-2 border border-green-600">Reference<sup>31</sup></td>
					</tr>
					<tr>
						<td class="p-2 border border-green-600">Human</td>
						<td class="p-2 border border-green-600"><i>PRRX1</i></td>
						<td class="p-2 border border-green-600"><i>PRRX1</i></td>
						<td class="p-2 border border-green-600">Reference<sup>41</sup></td>
					</tr>
				</tbody>
			</table>
			<p class="mb-2">
				Note the differences in gene and protein naming convention among commonly used model organisms in Table
				1.
			</p>
			<img class="mt-8" src="https://murawalalab.mdibl.org/wp-content/uploads/2021/04/Fig1-1024x433.jpg"
				alt="Figure 1 image">
			<p class="mt-8 mb-2">
				<b>Fig. 1. Gene nomenclature for duplicated and novel genes. </b>
				<b>a.</b> Two <i>Cat</i> loci arose by gene duplication. Comparison with the human genome reveals a
				single copy of <i>CAT</i> in the human genome. Thus, <i>Cat1</i> and <i>Cat2</i> are paralogs in the
				axolotl genome.
				<b>b.</b> Annotation of orthologs and paralogs. <b>Geneb1</b> is annotated based on its homology to
				<i>GENEB</i> in another organism (indicated by the pattern of exons), while a paralogous gene is
				annotated as <i>Geneb2</i> since it has a lower sequence similarity. Grey shaded area indicates the
				chromosome to highlight that <i>Geneb1</i> is in a different locus
				<b>c.</b> A putative novel gene family member in axolotl that has some (BLAST e-value 1e-16) similarity
				to a gene in another organism.
				<b>d.</b> A putative novel gene that does not have any homologs, but has a long (1941aa) open reading
				frame.
			</p>
			<h3 id="gene-nomenclature-in-axolotl" class="mt-8 mb-4 text-xl font-semibold">
				Gene nomenclature in axolotl
			</h3>
			<p class="mb-2">
				The chromosome-scale axolotl genome assemblies<sup>2,3</sup>
				made it possible to study not only the coding sequences, but also the evolution
				and synteny at the whole-genome level and accurate gene annotation is vital for
				those purposes. However, to ensure accurate and unambiguous communication
				between scientists, it is crucial to work out a set of rules for how the genes
				and proteins should be named, how to distinguish paralogs and orthologs, and how
				to name pseudogenes.
			</p>
			<ol class="list-decimal list-inside" start=4>
				<li class="mt-8">
					Orthologs and paralogs
					<p class="ml-6 mt-2">
						Unlike zebrafish and <i>Xenopus laevis</i>, the axolotl genome was not structured by
						whole-genome duplication events. However, axolotl-specific (i.e. in-paralogs) as
						well as salamander-specific (i.e. out-paralogs) gene duplicates are known (Fig.
						1a) and can be shared with other salamander taxa<sup>42</sup>. In order to
						define the evolutionary relationship between the genes, it is crucial to define
						whether the genes in different species derive from a common ancestor
						(orthologs), or whether the genes in the same species arose from the same
						ancestor (paralogs)<sup>43</sup>. While it is important to standardize the gene
						names for orthologous genes, it is also important to keep the gene naming
						compatible to that used across vertebrates, in order to make the analyses of the
						axolotl data comparable with the large body of available human and mouse
						datasets. Similar to the human and mouse orthologous gene nomenclature (HUMOT)
						project<sup>44</sup>, we propose to rely on a mixed approach that combines both
						the orthology information with the synteny and also integrates the data from
						expert orthology resources. The importance of the synteny can be demonstrated by
						a following example. Imagine a situation that in the axolotl, a gene homologous
						to a gene <i>GENEB</i> in another species was duplicated (Fig. 1b), while the copy
						that is more similar to the ancestral gene (indicated by the exon pattern in
						Fig. 1b) moved out of the locus. In this case, the copy that stayed in the locus
						should be annotated as <i>Geneb2</i>, while the one that moved out should be
						annotated as <i>Geneb1</i>. In contrast, if the copy that moved out is less similar
						to <i>GENEB</i> than the one that stayed, then the latter should be annotated as
						<i>Geneb1</i> and the former as <i>Geneb2</i>. If synteny is not conserved then
						phylogenetic analysis should be used to identify the relationships between the
						genes. As these relationships can be complex it is better to deliberately
						exclude ambiguous orthology information than to propagate incorrect assumptions
						about the genes.
					</p>
				</li>
			</ol>
			<p class="mt-8 mb-2">
				In the case of paralogous genes, one must be very cautious in order to avoid
				name collisions across species. Imagine, in the above example, there were also a
				gene <i>GENEB2</i> in the same species. However, this <i>GENEB2</i> is not orthologous to the
				newly annotated *Geneb2* in axolotl. In this case, we suggest using the next
				available number in the series, following consultation with other gene
				nomenclature groups. In the outlined example, it would be <i>Geneb3</i>. Hence, we
				always recommend contacting the axolotl gene nomenclature committee when naming a gene.
			</p>
			<p class="mb-2 pl-6">Paralogs:</p>
			<p class="mb-2 pl-6">Paired related homeobox 1 and Paired related homeobox 2: <i>Prrx1</i> and <i>Prrx2</i>
			</p>
			<p class="mb-2 pl-6">Duplicated genes:</p>
			<p class="mb-2 pl-6">Catalases – <i>Cat</i> and <i>Cat2</i></p>
			<ol class="list-decimal list-inside" start=5>
				<li class="mt-8">
					Novel genes.
					<p class="ml-6 mt-2">
						While paralogous genes should be treated as described above, potential novel
						genes should be annotated differently. Ideally, novel genes should be
						characterized functionally and named based on their function, whenever possible.
						However, for the vast majority of the novel genes this is not the case. We
						therefore suggest to examine the orthologous relationships to other functionally
						characterized or unambiguously annotated genes in axolotl or other organisms. If
						a novel gene can be assigned to a certain protein family but fails to fulfill
						the criteria outlined in 4 to be annotated unambiguously, we suggest adding a
						suffix ‘-like’ to the gene symbol of the closest ortholog based solely on the
						sequence homology. This annotation may be changed later when its existence is
						confirmed in the lab and the functional data become available. Finally, if none
						of those approaches can be applied, the novel gene should be annotated as
						<i>Locxxxx</i> (Fig. 1d), where xxxx are the NCBI gene IDs. In this case, the gene
						sequence should be submitted to NCBI first to get the NCBI gene ID, which
						ensures that this gene symbol is not arbitrary.
					</p>
					<p class="ml-6 mt-4 mt-2">Example:</p>
					<p class="ml-6 mt-2">Prothymosin-alpha-like : Prothymosin-alpha-like – Ptal (Fig. 1c) and</p>
					<p class="ml-6 mt-2">Loc12345 between Tmem79 and Smg5 (Fig. 1d). </p>
					<p class="ml-6 mt-8">5.1 Pseudogenes</p>
					<p class="ml-6 mt-2">
						Some genes may lose their open reading frame (ORF) in the course of the time
						and, thus, also their function. However, since the gene sequences are still
						present in the genome, they must be accurately annotated as they may have
						undesired effects on transgenics, transcript quantification and other analyses
						that rely on the gene sequence. To stay consistent with the nomenclature
						guidelines in other organisms and particularly in humans<sup>24</sup>, we suggest
						appending the suffix ‘p’ (pseudogene) followed by a number to the gene symbol of
						the ancestral gene if the gene is processed, while naming unprocessed genes as
						new members of the family with the suffix ‘p’.
					</p>
				</li>
				<li class="mt-8">
					Transcript variants
					<p class="ml-6 mt-2">
						We propose that the transcript variant names are formed after the following
						schema: GeneName.TranscriptNumber. All predicted transcripts of a gene are
						numbered by the order in which they were annotated. While in organisms with a
						well-established genome annotation, the nomenclature does not specifically deal
						with the transcript annotations, we feel that in axolotl the annotation is
						frequently changed at the moment and therefore it is vital to keep track of
						which isoforms were proven to be wrong. For example, imagine that a gene has
						three annotated isoforms, <i>Gene.1</i>, <i>Gene.2</i> and <i>Gene.3</i>.
						However, it turns out that <i>Gene.</i>3 is just an artifact. Nevertheless,
						another isoform, <i>Gene.4</i>, is shown to be very tissue-specific. At this
						point, it is better to have the isoforms annotated as <i>Gene.1</i>,
						<i>Gene.2</i> and <i>Gene.4</i>, to indicate that <i>Gene.3</i> does not exist and
						avoid confusion in case any works referred to <i>Gene.3</i> before it was excluded.
					</p>
					<p class="ml-6 mt-2">Example: transcript variants of <i>Prrx1</i></p>
					<p class="ml-6 mt-2"><i>Prrx1.1</i>, <i>Prrx1.2</i></p>
				</li>
				<li class="mt-8">
					Non-coding transcripts
					<p class="ml-6 mt-2">
						Similar to the guidelines for the human genome outlined in Seal et al, 2020<sup>45</sup>,
						we propose that non-coding transcripts are annotated according to their RNA
						type. MicroRNAs should be annotated as “mir-XXX”, where XXX is the submission
						ID in the miRbase database<sup>46</sup>. Transfer RNAs should receive gene names following
						the pattern tRNA-XXX-YYY- GtRNAdbID, where XXX is the three-letter amino acid
						code, YYY is the anticodon and GtRNAdbID is the gene ID in GtRNAdb database<sup>47</sup>.
						Other classes of non-coding RNAs should be named after consulting the gene
						naming committee as very little work has been done on non-coding RNAs in
						axolotl so far and, thus, the exact requirements will be met later.
					</p>
					<p class="ml-6 mt-2">
						Similar to the guidelines in other vertebrates, we suggest to use the gene
						symbol with the suffix ‘-as’ for non-coding transcripts that originate from the
						promoter as an annotated protein-coding gene on the opposite strand, e.g.
						<i>Dio3</i>-as for a non-coding RNA that originates from the <i>Dio3</i> promoter.
					</p>
				</li>
				<li class="mt-8">
					Mitochondrial genes
					<p class="ml-6 mt-2">
						In order to stay consistent with the well-annotated species, we propose to use
						the human annotation of the mitochondrial genes (NCBI Reference Sequence:
						NC_012920.1) in axolotl. However, in agreement with the remainder of the
						nomenclature, only the first letter should be capital, e.g. <i>Mtnd2</i> for the
						mitochondrially encoded NADH dehydrogenase 2, while the human counterpart would
						be <i>MTND2</i>.
					</p>
				</li>
			</ol>
			<h3 class="mt-8 mb-4 text-xl font-semibold">Chromosomes and aberrations</h3>
			<p class="mt-2">
				The axolotl genome is made up of ~32 Gb of DNA, which are distributed across 14
				chromosomes<sup>2,3,48</sup>. Chromosomes are numbered in a descending order based on
				their size, which was initially determined by meiotic mapping and recombination
				distances that define linkage groups<sup>37,49</sup>. Thus, as also seen in the human
				genome, chromosome ordering by size does not exactly correlate with ordering
				based upon chromosome base pair length. Further, axolotl chromosomes are
				divided into short and long arms via centromere. Conventionally, the short arm
				is called the <i>p</i> arm, while the long arm is called the <i>q</i> arm. For the sake of
				consistency, we propose to retain this nomenclature for the axolotl chromosome
				arms.
			</p>
			<ol class="list-decimal list-inside" start=9>
				<li class="mt-8">
					Deficiencies, duplication, inversion, insertion, translocation
					<p class="mt-2 ml-6">
						Chromosomal aberrations are known in every species and the axolotl is no
						exception. They can be mainly classified into deficiencies, duplication,
						inversion, insertions and translocations. Although a few chromosomal aberrations
						have been reported to date <sup>42</sup>, we anticipate that such annotations will arise in
						the near future from the analyses of the genome and transcriptome assemblies. We
						propose to use the following prefix for each of them which is in line with the
						usage of these terms by the zebrafish community <sup>29</sup>.
					</p>
					<p class="mt-2 ml-6">deficiencies, <i>Df</i></p>
					<p class="mt-2 ml-6">duplication, <i>Dp</i></p>
					<p class="mt-2 ml-6">inversion, <i>In</i></p>
					<p class="mt-2 ml-6">insertion, <i>Is</i></p>
					<p class="mt-2 ml-6">translocation, <i>T</i></p>
					<p class="mt-2 ml-6">Further, chromosome rearrangements are indicated with the following prefixes in
						italics, followed by the chromosome aberration details in parentheses and in italics, which in
						turn is followed by the name of the line in a regular font.</p>
					<p class="mt-2 ml-6"><i>Df(Chr#:xxx)</i>lineNN</p>
				</li>
			</ol>
			<h3 id="transgenic-lines-and-constructs" class="mt-8 mb-4 text-xl font-semibold">Transgenic lines and constructs</h3>
			<p class="mt-8">
				Since the first successful axolotl transgenics, which were ubiquitous
				fluorescent reporter expression lines<sup>6</sup>, transgenesis has made significant
				progress. In the last decade, a number of transgenic methodologies were
				successfully implemented in axolotl. This includes the I-SceI mediated
				transgenesis, Tol2-mediated transgenesis, TALEN-mediated transgenesis and
				CRISPR/Cas9-mediated transgenesis, all of which allow researchers to perform
				random transgenesis, generate knock-outs and knock-ins in axolotl<sup>5,7-10</sup>. With
				these advances, it is expected that more transgenic animals will be generated in
				the near future and a standard transgenic nomenclature is, thus, needed to
				assign the identifier information in a consistent and rigorous manner.
			</p>
			<ol class="list-decimal list-inside" start=10>
				<li class="mt-8">
					Random insertion
					<p class="mt-2 ml-6">
						I-SceI and Tol2- mediated transgenesis utilizes the flanking I-SceI restriction
						sites or Tol2 transposable elements to the cassette of interest. Co-injection of
						such construct with the I-SceI meganuclease or Tol2 mRNA/protein allows for
						random integration of the cassette of interest into the genome. We recommend
						highlighting such transgenic animals by the tg symbol followed by the method of
						transgenesis and name of the cassette in parentheses. We also propose separating
						regulatory elements (enhancer and promoters) and the coding sequence by colons.
						The name of the full cassette should be written in italics. If a foreign
						regulatory element is used for making the transgenic animal, then it should be
						mentioned as a one letter genus and three letter species symbol followed by a
						period in the nomenclature, such as from mouse – <i><u>M</u>.
							<u>mu</u>sculus</i> (Mmu). If the regulatory element of the axolotl is used then
						the species information can be omitted. Further, we recommend appending the
						developer’s lab code as a superscript after the parentheses to indicate the
						origin of the transgenic animal. In order to obtain the lab code, developer
						should register their lab/organization with international laboratory code
						registry (ILAR) (https://www.nationalacademies.org/ilar/lab-code-database).
					</p>
					<p class="mt-2 ml-6">
						tgSceI(<i>Mmu.Prrx1:GFPnls-T2a-ER<sup>T2</sup>-Cre-ER<sup>T2</sup></i>)<sup>Labcode</sup></p>
					<p class="mt-2 ml-6">Tol2 mediated transgenesis</p>
					<p class="mt-2 ml-6">tgTol2(<i>Mmu.Prrx1:GFPnls-T2a-Cre-ER<sup>T2</sup></i>)<sup>Labcode</sup></p>
					<p class="mt-2 ml-6">Often, transgenic animals are made with more than one cassette. In such
						instances, we suggest the use of semicolon (;) between two cassettes.</p>
					<p class="mt-2 ml-6">
						tgSceI(<i>Mmu.Prrx1:GFPnls-T2a-ER<sup>T2</sup>-Cre-ER<sup>T2</sup>;CAGGS:loxP-GFP-loxP-Cherry</i>)<sup>Labcode</sup>
					</p>
				</li>
				<li class="mt-8">
					Knockout lines
					<p class="mt-2 ml-6">
						Gene mutants are often generated with the aim to perform functional analysis.
						With the advent of CRISPR/Cas9, it has become relatively easy to generate such
						lines. Such germ-line transmitted targeted mutations<sup>&#8482; </sup> should be characterized
						for insertion or deletion and they should be annotated as I or D, respectively.
						In addition, the line name should also contain the position of the indel with
						respect to the start of the gene, the version of the genome and the number of
						nucleotides that are inserted or deleted. If the generated animals are
						heterozygous then the wildtype (+) should be mentioned as a second allele.
					</p>
					<p class="mt-2 ml-6">Example:</p>
					<p class="mt-2 ml-6">
						tm(<i>Prrx1<sup>153v6D8/153v6D8</sup></i>)<sup>Labcode</sup>, refers to a homozygous deletion of
						8 nucleotides at the nucleotide position 153 with respect to the start of <i>Prrx1</i> in the
						genome version 6.
					</p>
					<p class="mt-2 ml-6">
						tm(<i>Prrx1<sup>153v6I5/+</sup></i>)<sup>Labcode</sup>, refers to a heterozygous insertion of 5
						nucleotides at the nucleotide position 153 from the beginning of <i>Prrx1</i> in the genome
						version 6.
					</p>
				</li>
				<li class="mt-8">
					Knock-in lines
					<p class="mt-2 ml-6">
						Similarly, the names of a knock-in animal should contain the name of the gene
						locus where the transgene is inserted. At the moment only non-homologous
						end-joining (NHEJ) mediated knock-in possible<sup>8</sup> and such transgene knock-ins are
						generated by targeted mutation<sup>&#8482</sup> at either the N-terminus or the C-terminus of
						the endogenous ORF. This, in turn, may either retain or disrupt the native ORF.
						Similar to the cassette in the random transgenesis, regulatory elements and the
						coding DNA sequence should be separated by a colon and italicized. When a
						transgene is inserted at the C-terminus as a tag without disrupting the native
						coding sequence then it should be written as follows.
					</p>
					<p class="mt-2 ml-6">
						tm(<i>Prrx1</i><sup>t/+</sup><i>:Prrx1-T2a-Cherry</i>)<sup>Labcode</sup>, refers to a
						heterozygous knock-in at the <i>Prrx1</i> locus, which retains the native <i>Prrx1</i> gene
						structure and allows for tagging (t) at its C-terminus resulting in a fusion of <i>Prrx1
						</i>with T2a-Cherry. Similarly, a homozygous knock-in should be labeled as
						tm(<i>Prrx1</i><sup>t/t</sup><i>:Prrx1-T2a-Cherry</i>)<sup>Labcode</sup>.
					</p>
					<p class="mt-2 ml-6">
						However, the N-terminal insertion of a transgene via the NHEJ disrupts the native gene sequence.
						Hence, the N-terminus knock-ins are generated in one of the following two ways.
					</p>
					<p class="mt-2 ml-6">
						When the native coding sequence is disrupted leading to a heterozygous genotype,
					</p>
					<p class="mt-2 ml-6">
						tm(<i>Prrx1</i><sup>r/+</sup><i>:Cherry)</i><sup>Labcode</sup>, refers to
						the heterozygous N-terminus knock-in at the <i>Prrx1</i> locus, which replaces
						(r) the native gene with Cherry. In this situation, the native gene is not
						active and hence, these animals are heterozygous knock-outs for
						<i>Prrx1</i>.
					</p>
					<p class="mt-2 ml-6">
						When the native coding sequence is disrupted, but replaced with the cDNA of the
						native gene, which is also refered to as a mini-gene,
					</p>
					<p class="mt-2 ml-6">
						tm(<i>Prrx1</i><sup>r/+</sup><i>:miniPrrx1-T2a-Cherry)</i><sup>Labcode</sup>,
						refers to a heterozygous N-terminus knock-in at the <i>Prrx1</i> locus, which
						replaces (r) the native gene with a mini-gene version of <i>Prrx1</i>(cDNA),
						which is fused to T2a-Cherry. In this situation, since the native gene is
						replaced by the <i>Prrx1</i> cDNA, the animals are not considered as knock-outs
						for <i>Prrx1</i>.
					</p>
				</li>
			</ol>
			<p class="mt-8">
				Finally, we want to once again point out that the axolotl gene nomenclature
				committee should be contacted every time when naming a gene. We envisage that
				such systemic nomenclature would remove confusion among researchers and serve
				the entire community.
			</p>
			<h3 id="members-of-the-axolotl-gene-nomenclature-committee" class="mt-8 mb-4 text-xl font-semibold">Axolotl gene nomenclature committee:</h3>
			<ol class="list-decimal list-inside">
				<li class="mt-2">Dr. Sergej Nowoshilow, Research Institute of Molecular Pathology, Vienna, Austria</li>
				<li class="mt-2">
					<a class="text-axo-green" href="http://ibrr.scnu.edu.cn/a/20180516/123.html">Dr. Ji-Feng Fei</a>,
					South China Normal University, Guongzhou, China
				</li>
				<li class="mt-2">
					<a class="text-axo-green" href="http://www.vosslab.com/home.html">Dr. Randal S. Voss</a>, University
					of Kentucky, Lexington, USA
				</li>
				<li class="mt-2">
					<a class="text-axo-green" href="https://www.imp.ac.at/groups/elly-tanaka/">Dr. Elly M. Tanaka</a>,
					Research Institute of Molecular Pathology, Vienna, Austria
				</li>
				<li class="mt-2">
					<a class="text-axo-green" href="https://murawalalab.mdibl.org/">Dr. Prayag Murawala</a>, Mount
					Desert Island Biological Laboratory, Bar Harbor, USA
				</li>
			</ol>
			<p class="mt-8">
				If you would like to help us, please write an email to “To be announced”.
			</p>
			<h3 class="mt-8 mb-4 text-xl font-semibold">Advisor to the axolotl gene nomenclature committee:</h3>
			<p><a class="text-axo-green" href="https://www.ebi.ac.uk/about/people/elspeth-bruford">Dr. Elspeth
					Bruford</a>, EMBL – EBI, Cambridgeshire, UK</p>
			<h3 id="references" class="mt-8 mb-4 text-xl font-semibold">References</h3>
			<ol class="list-decimal list-inside">
				<li class="mt-8">Bryant DM, Johnson K, DiTommaso T, et al. A Tissue-Mapped Axolotl De Novo
					Transcriptome Enables Identification of Limb Regeneration Factors. Cell Rep.
					Jan 17 2017;18(3):762-776. <a class="text-axo-green"
						href="https://doi.org/10.1016/j.celrep.2016.12.063">https://doi.org/10.1016/j.celrep.2016.12.063</a>.
				</li>
				<li class="mt-2">Nowoshilow S, Schloissnig S, Fei JF, et al. The axolotl genome and the evolution
					of key tissue formation regulators. Nature. Feb 1 2018;554(7690):50-55.
					<a class="text-axo-green"
						href="https://doi.org/10.1038/nature25458">https://doi.org/10.1038/nature25458</a>.
				</li>

				<li class="mt-2">Smith JJ, Timoshevskaya N, Timoshevskiy VA, Keinath MC, Hardy D, Voss SR. A
					chromosome-scale assembly of the axolotl genome. Genome Res. Feb
					2019;29(2):317-324. <a class="text-axo-green"
						href="https://doi.org/10.1101/gr.241901.118">https://doi.org/10.1101/gr.241901.118</a>.
				</li>
				<li class="mt-2">
					Schloissnig S, Kawaguchi A, Nowoshilow S, et al. The giant axolotl genome
					uncovers the evolution, scaling, and transcriptional control of complex gene
					loci. Proc Natl Acad Sci U S A. Apr 13 2021;118(15). <a class="text-axo-green"
						href="https://doi.org/10.1073/pnas.2017176118">https://doi.org/10.1073/pnas.2017176118</a>.
				</li>
				<li class="mt-2">
					Khattak S, Murawala P, Andreas H, et al. Optimized axolotl (Ambystoma mexicanum)
					husbandry, breeding, metamorphosis, transgenesis and tamoxifen-mediated
					recombination. Nat Protoc. Mar 2014;9(3):529-40. <a class="text=axo-green"
						href="https://doi.org/10.1038/nprot.2014.040">https://doi.org/10.1038/nprot.2014.040</a>.
				</li>
				<li class="mt-2">
					Sobkow L, Epperlein HH, Herklotz S, Straube WL, Tanaka EM. A germline GFP
					transgenic axolotl and its use to track cell fate: dual origin of the fin
					mesenchyme during development and the fate of blood cells during regeneration.
					Dev Biol. Feb 15 2006;290(2):386-97. <a class="text-axo-green"
						href="https://doi.org/10.1016/j.ydbio.2005.11.037">https://doi.org/10.1016/j.ydbio.2005.11.037</a>.
				</li>
				<li class="mt-2">
					Woodcock MR, Vaughn-Wolfe J, Elias A, et al. Identification of Mutant Genes and
					Introgressed Tiger Salamander DNA in the Laboratory Axolotl, Ambystoma
					mexicanum. Sci Rep. Jan 31 2017;7(1):6. <a class="text-axo-green"
						href="https://doi.org/10.1038/s41598-017-00059-1">https://doi.org/10.1038/s41598-017-00059-1</a>.
				</li>
				<li class="mt-2">
					Fei JF, Lou WP, Knapp D, et al. Application and optimization of
					CRISPR-Cas9-mediated genome engineering in axolotl (Ambystoma mexicanum). Nat
					Protoc. Dec 2018;13(12):2908-2943. <a class="text-axo-green"
						href="https://doi.org/10.1038/s41596-018-0071-0">https://doi.org/10.1038/s41596-018-0071-0</a>.
				</li>
				<li class="mt-2">
					Fei JF, Schuez M, Tazaki A, Taniguchi Y, Roensch K, Tanaka EM. CRISPR-mediated
					genomic deletion of Sox2 in the axolotl shows a requirement in spinal cord
					neural stem cell amplification during tail regeneration. Stem Cell Reports. Sep
					9 2014;3(3):444-59. <a class="text-axo-green"
						href="https://doi.org/10.1016/j.stemcr.2014.06.018">https://doi.org/10.1016/j.stemcr.2014.06.018</a>.
				</li>
				<li class="mt-2">
					Flowers GP, Timberlake AT, McLean KC, Monaghan JR, Crews CM. Highly efficient
					targeted mutagenesis in axolotl using Cas9 RNA-guided nuclease. Development. May
					2014;141(10):2165-71. <a class="text-axo-green"
						href="https://doi.org/10.1242/dev.105072">https://doi.org/10.1242/dev.105072</a>.
				</li>
				<li class="mt-2">
					Fei JF, Schuez M, Knapp D, Taniguchi Y, Drechsel DN, Tanaka EM. Efficient gene
					knockin in axolotl and its use to test the role of satellite cells in limb
					regeneration. Proc Natl Acad Sci U S A. Nov 21 2017;114(47):12501-12506.
					<a class="text-axo-green"
						href="https://doi.org/10.1073/pnas.1706855114.">https://doi.org/10.1073/pnas.1706855114.</a>
				</li>

				<li class="mt-2">
					Gerber T, Murawala P, Knapp D, et al. Single-cell analysis uncovers convergence
					of cell identities during axolotl limb regeneration. Science. Oct 26
					2018;362(6413). <a class="text-axo-green"
						href="https://doi.org/10.1126/science.aaq0681.">https://doi.org/10.1126/science.aaq0681.</a>
				</li>

				<li class="mt-2">
					Khattak S, Sandoval-Guzman T, Stanke N, Protze S, Tanaka EM, Lindemann D. Foamy
					virus for efficient gene transfer in regeneration studies. BMC Dev Biol. May 3
					2013;13:17. <a class="text-axo-green"
						href="https://doi.org/10.1186/1471-213X-13-17.">https://doi.org/10.1186/1471-213X-13-17.</a>
				</li>

				<li class="mt-2">
					Oliveira CR, Lemaitre R, Murawala P, Tazaki A, Drechsel DN, Tanaka EM.
					Pseudotyped baculovirus is an effective gene expression tool for studying
					molecular function during axolotl limb regeneration. Dev Biol. Jan 15
					2018;433(2):262-275. <a class="text-axo-green"
						href="https://doi.org/10.1016/j.ydbio.2017.10.008.">https://doi.org/10.1016/j.ydbio.2017.10.008.</a>
				</li>

				<li class="mt-2">
					Roy S, Gardiner DM, Bryant SV. Vaccinia as a tool for functional analysis in
					regenerating limbs: ectopic expression of Shh. Dev Biol. Feb 15
					2000;218(2):199-205. <a class="text-axo-green"
						href="https://doi.org/10.1006/dbio.1999.9556.">https://doi.org/10.1006/dbio.1999.9556.</a>
				</li>

				<li class="mt-2">
					Whited JL, Tsai SL, Beier KT, et al. Pseudotyped retroviruses for infecting
					axolotl in vivo and in vitro. Development. Mar 2013;140(5):1137-46.
					<a class="text-axo-green"
						href="https://doi.org/10.1242/dev.087734.">https://doi.org/10.1242/dev.087734.</a>
				</li>

				<li class="mt-2">
					Masselink W, Reumann D, Murawala P, et al. Broad applicability of a streamlined
					ethyl cinnamate-based clearing procedure. Development. Feb 1 2019;146(3).
					<a class="text-axo-green"
						href="https://doi.org/10.1242/dev.166884.">https://doi.org/10.1242/dev.166884.</a>
				</li>

				<li class="mt-2">
					Pende M, Vadiwala K, Schmidbaur H, et al. A versatile depigmentation, clearing,
					and labeling method for exploring nervous system diversity. Sci Adv. May
					2020;6(22):eaba0365. <a class="text-axo-green"
						href="https://doi.org/10.1126/sciadv.aba0365.">https://doi.org/10.1126/sciadv.aba0365.</a>
				</li>

				<li class="mt-2">
					Duerr TJ, Comellas E, Jeon EK, et al. 3D visualization of macromolecule
					synthesis. Elife. Oct 14 2020;9. <a class="text-axo-green"
						href="https://doi.org/10.7554/eLife.60354.">https://doi.org/10.7554/eLife.60354.</a>
				</li>

				<li class="mt-2">
					Adrados CS, Yu Q, Bolanos Castro LA, Rodriguez Cabrera LA, Yun MH.
					Salamander-Eci: An optical clearing protocol for the three-dimensional
					exploration of regeneration. Dev Dyn. Oct 20 2020.
					<a class="text-axo-green"
						href="https://doi.org/10.1002/dvdy.264.">https://doi.org/10.1002/dvdy.264.</a>
				</li>
				<li class="mt-2">
					Davisson MT. Rules and guidelines for nomenclature of mouse genes. International
					Committee on Standardized Genetic Nomenclature for Mice. Gene. Sep 30
					1994;147(2):157-60. <a class="text-axo-green"
						href="https://doi.org/10.1016/0378-1119(94)90060-4">https://doi.org/10.1016/0378-1119(94)90060-4</a>.
				</li>

				<li class="mt-2">
					Dunn LCG, H. Snell, G. D. . Report of the committee on mouse genetics
					nomenclature. Journal of Heredity. December 1940 1940;31(12):505-506.
					<a class="text-axo-green"
						href="https://doi.org/10.1093/oxfordjournals.jhered.a104827">https://doi.org/10.1093/oxfordjournals.jhered.a104827</a>.
				</li>

				<li class="mt-2">
					Wood R. Trends in Genetics: Genetic nomenclature guide with information on
					websites.: Cambridge, UK : Elsevier Trends Journals; 1998.
				</li>

				<li class="mt-2">
					Bruford EA, Braschi B, Denny P, Jones TEM, Seal RL, Tweedie S. Guidelines for
					human gene nomenclature. Nat Genet. Aug 2020;52(8):754-758.
					<a class="text-axo-green"
						href="https://doi.org/10.1038/s41588-020-0669-3">https://doi.org/10.1038/s41588-020-0669-3</a>.
				</li>

				<li class="mt-2">
					Shows TB, Alper CA, Bootsma D, et al. International system for human gene
					nomenclature (1979) ISGN (1979). Cytogenet Cell Genet. 1979;25(1-4):96-116.
					<a class="text-axo-green"
						href="https://doi.org/10.1159/000131404">https://doi.org/10.1159/000131404</a>.
				</li>

				<li class="mt-2">
					Xenopus_Gene_Nomenclature_Committee. Gene Nomenclature Guidelines
					https://wwwxenbaseorg/gene/static/geneNomenclaturejsp 2013.
				</li>

				<li class="mt-2">
					Karimi K, Fortriede JD, Lotay VS, et al. Xenbase: a genomic, epigenomic and
					transcriptomic model organism database. Nucleic Acids Res. Jan 4
					2018;46(D1):D861-D868. <a class="text-axo-green"
						href="https://doi.org/10.1093/nar/gkx936">https://doi.org/10.1093/nar/gkx936</a>.
				</li>

				<li class="mt-2">
					Ruzicka L, Howe DG, Ramachandran S, et al. The Zebrafish Information Network:
					new support for non-coding genes, richer Gene Ontology annotations and the
					Alliance of Genome Resources. Nucleic Acids Res. Jan 8 2019;47(D1):D867-D873.
					<a class="text-axo-green"
						href="https://doi.org/10.1093/nar/gky1090">https://doi.org/10.1093/nar/gky1090</a>.
				</li>

				<li class="mt-2">
					Zebrafish_Nomenclature_Committee. ZFIN Zebrafish Nomenclature Conventions
					https://wikizfinorg/display/general/ZFIN+Zebrafish+Nomenclature+Conventions
					2019.
				</li>

				<li class="mt-2">
					Horvitz HR, Brenner S, Hodgkin J, Herman RK. A uniform genetic nomenclature for
					the nematode Caenorhabditis elegans. Mol Gen Genet. Sep 1979;175(2):129-33.
					<a class="text-axo-green"
						href="https://doi.org/10.1007/BF00425528">https://doi.org/10.1007/BF00425528</a>.
				</li>

				<li class="mt-2">
					Burt DW, Carre W, Fell M, et al. The Chicken Gene Nomenclature Committee report.
					BMC Genomics. Jul 14 2009;10 Suppl 2:S5.
					<a class="text-axo-green"
						href="https://doi.org/10.1186/1471-2164-10-S2-S5">https://doi.org/10.1186/1471-2164-10-S2-S5</a>.
				</li>
				<li class="mt-2">
					Kusumi K, Kulathinal RJ, Abzhanov A, et al. Developing a community-based genetic
					nomenclature for anole lizards. BMC Genomics. Nov 11 2011;12:554.
					<a class="text-axo-green"
						href="https://doi.org/10.1186/1471-2164-12-554">https://doi.org/10.1186/1471-2164-12-554</a>.
				</li>

				<li class="mt-2">
					Stolfi A, Sasakura Y, Chalopin D, et al. Guidelines for the nomenclature of
					genetic elements in tunicate genomes. Genesis. Jan 2015;53(1):1-14.
					<a class="text-axo-green"
						href="https://doi.org/10.1002/dvg.22822">https://doi.org/10.1002/dvg.22822</a>.
				</li>

				<li class="mt-2">
					Reddien PW, Newmark PA, Sanchez Alvarado A. Gene nomenclature guidelines for the
					planarian Schmidtea mediterranea. Dev Dyn. Nov 2008;237(11):3099-101.
					<a class="text-axo-green"
						href="https://doi.org/10.1002/dvdy.21623">https://doi.org/10.1002/dvdy.21623</a>.
				</li>

				<li class="mt-2">
					Smith JJ, Voss SR. Gene order data from a model amphibian (Ambystoma): new
					perspectives on vertebrate genome structure and evolution. BMC Genomics. Aug 29
					2006;7:219. <a class="text-axo-green"
						href="https://doi.org/10.1186/1471-2164-7-219">https://doi.org/10.1186/1471-2164-7-219</a>.
				</li>

				<li class="mt-2">
					Voss SR, Kump DK, Putta S, et al. Origin of amphibian and avian chromosomes by
					fission, fusion, and retention of ancestral chromosomes. Genome Res. Aug
					2011;21(8):1306-12. <a class="text-axo-green"
						href="https://doi.org/10.1101/gr.116491.110">https://doi.org/10.1101/gr.116491.110</a>.
				</li>

				<li class="mt-2">
					Voss SR, Smith JJ, Gardiner DM, Parichy DM. Conserved vertebrate chromosome
					segments in the large salamander genome. Genetics. Jun 2001;158(2):735-46.
				</li>

				<li class="mt-2">
					Wilkinson MD, Dumontier M, Aalbersberg IJ, et al. The FAIR Guiding Principles
					for scientific data management and stewardship. Sci Data. Mar 15 2016;3:160018.
					<a class="text-axo-green"
						href="https://doi.org/10.1038/sdata.2016.18">https://doi.org/10.1038/sdata.2016.18</a>.
				</li>

				<li class="mt-2">
					Joven A, Elewa A, Simon A. Model systems for regeneration: salamanders.
					Development. Jul 22 2019;146(14). <a class="text-axo-green"
						href="https://doi.org/10.1242/dev.167700">https://doi.org/10.1242/dev.167700</a>.
				</li>

				<li class="mt-2">
					International_Committee_on_Standardized_Genetic_Nomenclature_for_Mice.
					Guidelines for Nomenclature of Genes, Genetic Markers, Alleles, and Mutations in
					Mouse and Rat http://wwwinformaticsjaxorg/mgihome/nomen/geneshtml 2018.
				</li>

				<li class="mt-2">
					HUGO_Gene_Nomenclature_Committee. HGNC Guidelines [WWW Document]. HGNC.
					https://wwwgenenamesorg/about/guidelines/. 2020.
				</li>

				<li class="mt-2">
					Keinath MC, Timoshevskaya N, Timoshevskiy VA, Voss SR, Smith JJ. Miniscule
					differences between sex chromosomes in the giant genome of a salamander. Sci
					Rep. Dec 14 2018;8(1):17882. <a class="text-axo-green"
						href="https://doi.org/10.1038/s41598-018-36209-2">https://doi.org/10.1038/s41598-018-36209-2</a>
				</li>
				<li class="mt-2">
					Koonin EV. An apology for orthologs – or brave new memes. Genome Biol.
					2001;2(4):COMMENT1005. <a class="text-axo-green"
						href="https://doi.org/10.1186/gb-2001-2-4-comment1005">https://doi.org/10.1186/gb-2001-2-4-comment1005</a>.
				</li>

				<li class="mt-2">
					Wright MW, Bruford EA. Human and orthologous gene nomenclature. Gene. Mar 15
					2006;369:1-6. <a class="text-axo-green"
						href="https://doi.org/10.1016/j.gene.2005.10.029">https://doi.org/10.1016/j.gene.2005.10.029</a>.
				</li>

				<li class="mt-2">
					Seal RL, Chen LL, Griffiths-Jones S, et al. A guide to naming human non-coding
					RNA genes. EMBO J. Mar 16 2020;39(6):e103777.
					<a class="text-axo-green"
						href="https://doi.org/10.15252/embj.2019103777">https://doi.org/10.15252/embj.2019103777</a>.
				</li>

				<li class="mt-2">
					Ambros V, Bartel B, Bartel DP, et al. A uniform system for microRNA annotation.
					RNA. Mar 2003;9(3):277-9. <a class="text-axo-green"
						href="https://doi.org/10.1261/rna.2183803">https://doi.org/10.1261/rna.2183803</a>.
				</li>

				<li class="mt-2">
					Chan PP, Lowe TM. GtRNAdb 2.0: an expanded database of transfer RNA genes
					identified in complete and draft genomes. Nucleic Acids Res. Jan 4
					2016;44(D1):D184-9. <a class="text-axo-green"
						href="https://doi.org/10.1093/nar/gkv1309">https://doi.org/10.1093/nar/gkv1309</a>.
				</li>

				<li class="mt-2">
					Callan HG. Chromosomes and nucleoli of the axolotl, Ambystoma mexicanum. J Cell
					Sci. Mar 1966;1(1):85-108.
				</li>

				<li class="mt-2">
					Smith JJ. Kump DK, Walker JA, Parichy DM, Voss, SR. A comprehensive expressed
					sequence tag linkage map for tiger salamander and Mexican axolotl: enabling gene
					mapping and comparative genomics in Ambystoma. Genetics. Nov 2005;
					171(3):1161-71. <a class="text-axo-green"
						href="https://doi.org/10.1534/genetics.105.04643">https://doi.org/10.1534/genetics.105.046433</a>
				</li>
			</ol>
		</div>
	</div>
</div>